### ISSUE TITLE
Support for resumable uploads

### ISSUE THREAD

**Issue by heyzooi** [uploadcare/uploadcare-ios#24]
Support for resumable uploads

**Comment by dmitry-mukhin** [comment]
Can you please elaborate?
At the moment Upload API does not support this, so, probably, it's impossible to do that in the client.
But I want to know what you want here specifically.

**Comment by heyzooi** [comment]
Using the `Content-Range` HTTP header the client could ask the server how much of a file was stored, so the client could send only the missing part of the file, instead of try send the entire file again. This is very important for real case scenarios on mobile devices where the network is sometimes unstable (losing many packages) or slow quite often.

**Comment by heyzooi** [comment]
Yeah, i know the server does not support. I tried to implement in the client, but I would need support for this feature in the server side.

**Comment by heyzooi** [comment]
This also could be a way to upload big files in chunks.

**Comment by rsedykh** [comment]
@heyzooi jfyi, I know it's maybe way past late, but we'll try to do it in [uploadcare-swift](https://github.com/uploadcare/uploadcare-swift/issues/57). :-)
